### PR TITLE
[SPARK-6851] [SQL] function least/greatest follow up

### PR DESCRIPTION
**Pull request by adrian-wang**
This is a follow up of remaining comments from #6851

### CODE REVIEW

**Comment by adrian-wang** [comment]
retest this please.

File: sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/conditionals.scala [hunk 1]
```diff
@@ -311,7 +311,11 @@ case class CaseKeyWhen(key: Expression, branches: Seq[Expression]) extends CaseW
   }
 }
 
-case class Least(children: Expression*) extends Expression {
+/**
+ * A function that returns the least value of all parameters.
```
Comment (rxin) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/conditionals.scala]: i think it's also important to explicitly say that "least value of all parameters, skipping null values".

**Comment by rxin** [comment]
Thanks - merging this.

**Comment by rxin** [comment]
@adrian-wang can you close this ticket? I believe it has been merged already but github is having issue with pr closure.

**Pull request closed without merging**